ref/src/yppm_driver.f90

```diff
@@ -106,8 +123,8 @@ program yppm_driver
   call write_state(TRIM(output_file))
 
   ! Write timing information
-  write(*,*)
-  write(*,'(A,F12.9)') "Total time (in seconds)=" ,  ((count_end - count_start) * 1.0) / count_rate
+  write (*, *)
+  write (*, '(A,F12.9)') "Total time (in seconds)=", ((count_end - count_start)*1.0)/count_rate
```

Comment:
Removal of spaces around mathematical operators causes style inconsistent with previous code and with changes made below which inserts such spaces.  Spaces are preferred for increased legibility.

Reply:
This is another Prettify change.  I'm assuming you want the expressions left alone, do not put in the spaces, yes? no?

Reply:
No.  Leave the spaces in.  Your code removes them, which makes it inconsistent with the rest of the code's style and makes it less readable.

Reply:
To clarify a bit more, I am talking about keeping the spaces around the mathematical operators and removing the spaces before the open parentheses. Take a look at [yppm_core.F90](https://github.com/NOAA-GSL/SENA-yppm/blob/develop/ref/src/yppm_core.F90) and make this line consistent with the style in there:

```
write(*, *)
write(*, '(A, F12.9)') "Total time (in seconds)=", ((count_end - count_start) * 1.0) / count_rate
```